src/styles/variables.json

```diff
@@ -3,6 +3,7 @@
     "black": "#000",
     "countBadgeBackground": "#e9163c",
     "bdl-box-blue": "#0061d5",
+    "bdl-box-blue-110": "#0057C0",
```

Comment:
What is blue-110? Is this approved by design?

Reply:
Yea it's in the figma design.

Reply:
I removed all of these changes. I pulled them it from BP instead.